app/controllers/application_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ApplicationController < ActionController::Base
```

Comment:
this was moved from the radstack project, I confirmed all other apps had the minimal version of this file without the redirect_root_domain action, which isn't active w/o the marketing site config setting